doc/api/icu.md

```diff
@@ -0,0 +1,65 @@
+# icu
+
+    Stability: 1 - Experimental
+
+The `icu` module provides an interface to Unicode and Internationalization
+functionality provided by the ICU4C library.
+
+```js
+const icu = require('icu');
+```
+
+## icu.transcode(buf, from_enc, to_enc)
+
+* `buf` {Buffer} A `Buffer` instance
+* `from_enc` {string} The current encoding
+* `to_enc` {string} The target encoding
+
+Re-encodes the given `Buffer` from one character encoding to another. Returns
+a new `Buffer` instance.
+
+## icu.codePointAt(buf, offset, encoding)
+
+* `buf` {Buffer} A `Buffer` instance
+* `offset` {integer} The offset position
```

Comment:
I still think these should have clarified that `offset` is measured in bytes, not characters, even if it’s just saying `byte offset`.